src/main/java/com/google/devtools/build/lib/vfs/FileSystemUtils.java

```diff
@@ -844,16 +844,37 @@ public static String readContent(Path inputFile, Charset charset) throws IOExcep
   }
 
   /**
-   * Reads at most {@code limit} bytes from {@code inputFile} and returns it as a byte array.
+   * Reads at most {@code limit} bytes from {@code inputFile} and returns them as a byte array.
    *
    * @throws IOException if there was an error.
    */
   public static byte[] readContentWithLimit(Path inputFile, int limit) throws IOException {
```

Comment:
Within Bazel, this function is now only called by tests. I can get rid of it if that's also the case for Blaze.

Reply:
It's not used anywhere else internally. I guess we can get rid of it.